modules/ROOT/pages/_partials/gs-accessing-prometheus-grafana-proc.adoc

```diff
@@ -0,0 +1,18 @@
+[id='gs-accessing-prometheus-grafana-proc']
+
+ifdef::env-github[]
+:imagesdir: ../images/
+endif::[]
+
+= Accessing Prometheus and Grafana
+
+.Prerequisite
+* You have logged in to the OpenShift console as the `customer-admin` user.
```

Comment:
This might be better as step 1. If not, the header should be plural (Prerequisites).

Reply:
I know what you mean. The reason I did it this way is that there are already multiple instances in the 1.7 docs of 
`.Prerequisite
* You have logged in to the ...... 
`
so I was being consistent with this styling. Will keep it as it is until we decide what to do with this styling overall.